pkgs/build-support/bintools-wrapper/default.nix

```
@@ -192,7 +192,7 @@ stdenv.mkDerivation {
       unset ldInner
     '') + ''
 
-      for variant in ld.gold ld.bfd ld.lld; do
+      for variant in ld.gold ld.bfd ld.lld ld.mold; do
```

```suggestion
      for variant in ld.*; do
```
Is it time to do this instead?

---

It's time to do so! Doing this also makes convenience for more new linkers.

---

OK then apply that that diff / squash history as you see fit!